YPP-0034.md

```diff
@@ -0,0 +1,297 @@
+3. Deploy the FYFRAX2209 and FYFRAX2303 fyToken & pools (no need for governance approval)
+4. Deploy YSFRAX1YEV and YSFRAX1YOD strategies (no need for governance approval)
+5. Deploy OnChainTest (no need for governance approval)
+6. Governance proposal
+   a. Update chi and rate in accumulator oracle
+   b. Update Chainlink sources
+   c. Update sources in the Composite Oracle
+   d. Update Paths in the Composite Oracle
+   e. Add FRAX as an asset
+   f. Make FRAX a base
+   g. Approve DAI, USDC, WBTC, WSTETH, ENS and LINK as collaterals for FRAX using the Composite Oracle
+   h. Approve ETH as collateral for FRAX using the Chainlink Oracle
+   i. Add the FYFRAX2209 and FYFRAX2303 fyToken and pools as series
+   j. Initialize the FYETH2206 and FYETH2209 pools with 100 FRAX each, and burn the LP tokens
+   k. Orchestrate the YSETH6MMS and YSETH6MJD strategies
+   l. Initialize the YSETH6MMS and YSETH6MJD strategies with 100 FRAX each, and burn the strategy tokens
+
+These steps in 6 are encoded into the [addFraxSeries.ts](https://github.com/yieldprotocol/environments-v2/blob/aeb8eb9d8f45052103f0514adef129616c27ebea/scripts/governance/addSeries/addFraxSeries/addFraxSeries.ts) scripts, while the steps 1 to 5 are the regular deployment scripts from the [environment deployment](https://github.com/yieldprotocol/environments-v2/tree/feat/eth-series/scripts/governance/redeploy).
+
+[This input](https://github.com/yieldprotocol/environments-v2/blob/aeb8eb9d8f45052103f0514adef129616c27ebea/scripts/governance/addSeries/addFraxSeries/addFraxSeries.mainnet.config.ts) will be used:
+
+## Proposal Config
+
+```
+export const rateChiSources: Array<[string, string, string, string]> = [
+  [FRAX, RATE, WAD.toString(), WAD.toString()],
+  [FRAX, CHI, WAD.toString(), WAD.toString()],
+]
+
+// Assets that will be made into a base
+export const bases: Array<[string, string]> = [[FRAX, newJoins.get(FRAX) as string]]
+
+export const assetsToAdd: Map<string, string> = new Map([[FRAX, assets.get(FRAX) as string]])
+
+export const chainlinkSources: Array<[string, string, string, string, string]> = [
+  [FRAX, assets.get(FRAX) as string, ETH, assets.get(ETH) as string, '0x14d04Fff8D21bd62987a5cE9ce543d2F1edF5D3E'],
+]
+
+export const newChiSources: Array<[string, string]> = [[FRAX, protocol.get(ACCUMULATOR) as string]]
+
+export const newRateSources: Array<[string, string]> = [[FRAX, protocol.get(ACCUMULATOR) as string]]
+
+export const compositeSources: Array<[string, string, string]> = [
+  [FRAX, ETH, protocol.get(CHAINLINK) as string],
+]
+
+export const newCompositePaths: Array<[string, string, Array<string>]> = [
+  [FRAX, ENS, [ETH]],
+  [FRAX, WSTETH, [ETH, STETH]],
+]
+
+// Input data: baseId, ilkId, ratio (1000000 == 100%), line, dust, dec
+export const newChainlinkLimits: Array<[string, string, number, number, number, number]> = [
+  [FRAX, FRAX, 1000000, 5000000, 5000, 18],
+  [FRAX, ETH, 1400000, 1000000, 5000, 18],
+  [FRAX, DAI, 1330000, 1000000, 5000, 18],
```

Comment:
For FRAX/DAI and FRAX/USD, we can probably be way less strict on collateralization ratio. Let's do 110%